CHANGELOG.rst

```diff
@@ -15,6 +15,11 @@ Changelog
   :meth:`~cryptography.hazmat.primitives.asymmetric.ec.EllipticCurvePublicKey.verify`
   has always been to check whether or not
   :class:`~cryptography.exceptions.InvalidSignature` was raised.
+* Accessing a critical extension on an X.509 object will no longer raise an
```

Comment:
"Accessing an unrecognized extension marked critical on an X.509 object" reads better to me -- the current verbiage makes it sound like all extensions marked critical raised the exception, rather than just the ones we didn't know how to process.